app/models/journeys/session.rb

```diff
@@ -65,5 +65,11 @@ def tps_school_for_student_loan_in_previous_financial_year
           teacher_reference_number: answers.teacher_id_user_info["trn"]
         )
     end
+
+    def load_answers(other_answers)
+      other_answers.attributes.each do |field,value|
+        answers.public_send("#{field}=", value)
+      end
+    end
```

Comment:
i added this so i can instantiate the class and load in some custom values as i need to be able to pass certain values to `EligibilityChecker` to determine the outcome. Not sure if there is a better way to do this?

Reply:
this has now been removed in favour of using factories in specs